AppCenterCrashes/AppCenterCrashes/MSCrashes.mm

```diff
@@ -147,10 +152,6 @@ @interface MSCrashes ()
  * @warning This property only has an updated value once the SDK has been properly initialized!
  */
 @property BOOL didReceiveMemoryWarningInLastSession;
-/**
- * Indicates if the delayedProcessingSemaphore will need to be released anymore. Useful for preventing overflows.
- */
-@property BOOL shouldReleaseProcessingSemaphore;
```

Comment:
Any reasons for removing it? Just replacing it by once token?

Reply:
Yes, once token is easier to read and understand